integrationtesting/opendcs-tests/src/test/java/org/opendcs/dao/CwmsLocationLevelDAOTestIT.java

```diff
@@ -0,0 +1,361 @@
+import org.opendcs.utils.logging.OpenDcsLoggerFactory;
+import org.slf4j.Logger;
+
+/**
+ * Integration tests for CwmsLocationLevelDAO
+ * These tests require a real CWMS database connection
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CwmsLocationLevelDAOTestIT extends AppTestBase
+{
+    private static final Logger log = OpenDcsLoggerFactory.getLogger();
+
+    @ConfiguredField
+    private TimeSeriesDb tsDb;
+    
+    @ConfiguredField
+    private OpenDcsDatabase db;
+
+    private LocationLevelDAI dao;
+    private static boolean testDataLoaded = false;
+    
+    @BeforeAll
+    public void setupTestData() throws Exception
+    {
+        if (tsDb instanceof CwmsTimeSeriesDb && !testDataLoaded)
+        {
+            try
+            {
+                log.info("Loading location level test data for integration tests");
+                loadLocationLevelTestData();
+                testDataLoaded = true;
+                log.info("Location level test data loaded successfully");
+            }
+            catch (Exception ex)
+            {
+                log.error("Failed to load test data: " + ex.getMessage(), ex);
+                throw ex;
+            }
+        }
+    }
+    
+    @BeforeEach
+    public void setUp() throws Exception
+    {
+        if (tsDb instanceof CwmsTimeSeriesDb)
+        {
+            Optional<LocationLevelDAI>  dai = db.getDao(LocationLevelDAI.class);
+            assertTrue(dai.isPresent(), "Unable to retrieve LocationLevelDAI instance from database.");
+            dao = dai.get();
+        }
+        else
+        {
+            // For non-CWMS databases, skip these tests
+            dao = null;
+        }
+    }
+    
+    @Test
+    @EnableIfTsDb({"CWMS-Oracle"})
+    public void testGetLatestLocationLevelValue_Basic() throws Exception
+    {
+        if (dao == null) 
+        {
+            return; // Skip for non-CWMS databases
+        }
+        
+        // This test verifies the method executes without error
+        // Test data is created during container initialization
+        String testLocationLevelId = "TEST_LOCATION.Stage.Const.0.Test";
+        
+        try (DataTransaction tx = db.newTransaction())
+        {
+            LocationLevelValue value = dao.getLatestLocationLevelValue(tx, testLocationLevelId);
+            
+            // Test passes whether data exists or not
+            // If value is not null, verify its structure
+            if (value != null)
+            {
+                assertNotNull(value.getLevelDate(), "Level date should not be null when value exists");
+                assertNotNull(value.getUnits(), "Units should not be null when value exists");
+                assertEquals(testLocationLevelId, value.getLocationLevelId(), 
+                    "Location level ID should match request");
+                
+                System.out.println("Found test data for location: " + testLocationLevelId);
```

Comment:
Use logger.

NOTE, slf4j *should* also be used in tests, but java util logging is fine as well.